test/server_integration/http/platform/headers.ts

```diff
@@ -74,7 +74,7 @@ export default function ({ getService }: FtrProviderContext) {
       }
 
       await performRequest();
-      const defaultHeadersTimeout = 40 * oneSec;
+      const defaultHeadersTimeout = 60 * oneSec;
```

Comment:
changed to 60sec in nodejs v12 https://github.com/nodejs/node/pull/30071

Although we can remove this test if we are okay to rely on nodejs, I'd prefer to have it to make sure this functionality is not broken.